File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    
    for i in range(0, len(t)-len(p)+1):
        
        sliced = ""
        
        for j in range(i, i+len(p)):
            sliced += t[j]
            
        if sliced <= p:
            answer += 1
        
    return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.java
class Solution {
    public int solution(String t, String p) {
        int answer = 0;
        
        
        for(int i = 0; i < t.length()-p.length()+1; i++){
            String div = "";
            for(int j = i; j < i+p.length(); j++){
                div += t.charAt(j);
            }
            if( Long.parseLong(div) <= Long.parseLong(p)){
                answer += 1;
            }
        }
        return answer;
    }
}